testutils/networksuite/networksuite.go

```diff
@@ -27,13 +33,43 @@ import (
 	blocksdktypes "github.com/skip-mev/block-sdk/x/blocksdk/types"
 )
 
-// NetworkTestSuite is a test suite for tests that initializes a network instance.
+var (
+	chainID = "chain-" + cmtrand.NewRand().Str(6)
+
+	DefaultAppConstructor = func(val network.ValidatorI) servertypes.Application {
+		return app.New(
+			log.NewLogger(os.Stdout),
+			dbm.NewMemDB(),
+			nil,
+			true,
+			simtestutil.EmptyAppOptions{},
+			baseapp.SetPruning(pruningtypes.NewPruningOptionsFromString(val.GetAppConfig().Pruning)),
+			baseapp.SetMinGasPrices(val.GetAppConfig().MinGasPrices),
+			baseapp.SetChainID(chainID),
+		)
+	}
+
+	genBalance = sdk.NewCoin(sdk.DefaultBondDenom, math.NewInt(1000000000000000000))
+
+	txc client.TxConfig
```

Comment:
Very similar to what I pushed before in the SetupSuite function. Let's either use/update that stuff or move it up here and delete what's in that function. Doesn't make much difference to me where it gets set, but let's not have it in two places.